sdk/cosmos/azure-cosmos/CHANGELOG.md

```diff
@@ -1,27 +1,20 @@
 ## Release History
 
-### 4.32.0-beta.1 (Unreleased)
-
-#### Features Added
-
-#### Breaking Changes
-
-#### Bugs Fixed
-
+### 4.32.0 (2022-06-27)
 #### Other Changes
 * Remove requires `io.netty.transport.epoll` from `module-info`.
 * Converted from `durationInMicroSec` to `durationInMilliSecs` in `CosmosDiagnostics` - See [PR 29643](https://github.com/Azure/azure-sdk-for-java/pull/29643)
 
 ### 4.31.0 (2022-06-08)
-
+> [!IMPORTANT]
+> We strongly recommend our customers to use version 4.31.0 and above.
```

Comment:
We need a better/more convincing explanation why we recommend this version, such as list some of the critical bug fixes/vulnerabilities that were addressed :-)

Reply:
Got it, will update the changelog with some pointers on these critical fixes / vulnerabilities.